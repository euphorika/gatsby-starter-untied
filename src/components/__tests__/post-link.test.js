import React from 'react'
import renderer from 'react-test-renderer'
import PostLink from '../post-link'

const postLinkEntry = {
  excerpt: 'excerpt',
  frontmatter: {
    title: 'title',
    date: 'November 05, 2018',
  },
  fields: {
    slug: '/slug',
  },
}

describe('Post-Link', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<PostLink post={postLinkEntry} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
