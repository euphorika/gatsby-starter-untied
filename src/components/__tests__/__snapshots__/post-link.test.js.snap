// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post-Link renders correctly 1`] = `
<div>
  <h2>
    <Link
      to="/slug"
    >
      title
       (
      November 05, 2018
      )
    </Link>
  </h2>
  <p>
    excerpt
  </p>
</div>
`;
